pymobiledevice3/bonjour.py

```diff
@@ -104,6 +104,8 @@ def get_ipv4_addresses() -> List[str]:
     ips = []
     for adapter in get_adapters():
         for ip in adapter.ips:
+            if "tun" in ip.ip:
```

Comment:
the `ip` is a simple ip, you should check `adapter.nice_name` instead, as u already did in the ipv6 implementation